notebooks/basics/single-systems.ipynb

```diff
@@ -183,27 +178,26 @@
    "id": "936e5686",
    "metadata": {},
    "source": [
-    "### 1.2 Measuring classical states <a id='classical-measure'></a>\n",
+    "### 1.2 Measuring probabilistic states <a id='classical-measure'></a>\n",
     "\n",
-    "Now let us briefly consider what happens if we *measure* a system when it is in a probabilistic state.\n",
-    "By measuring a system, we mean that we look at the system and unambiguously recognize whatever classical state it is in.\n",
+    "Now let us briefly consider what happens if we *measure* a system when it is in a probabilistic state. By measuring a system, we mean that we look at the system and unambiguously recognize whatever classical state it is in. Intuitively, we can never \"see\" a system in a probabilistic state; a measurement will yield exactly one of the allowed classical states.\n",
     "\n",
     "Measurement changes our knowledge of the system, and therefore changes the probabilistic state that we associate with that system: if we recognize that $\\mathsf{X}$ is in the classical state $a\\in\\Sigma$, then the new probability vector representing our knowledge of $\\mathsf{X}$ becomes a vector having a $1$ in the entry corresponding to $a$ and $0$ for all other entries.\n",
     "This vector indicates that $\\mathsf{X}$ is in the classical state $a$ with certainty, which we know having just recognized it.\n",
     "\n",
     "We denote the vector just described, meaning the vector having a $1$ in the entry corresponding to $a$ and $0$ for all other entries, by $\\vert a\\rangle.$\n",
-    "This vector is read as \"ket $a$\" for a reason that will be explained later.\n",
+    "This vector is read as \"ket $a$\" for a reason that will be explained shortly.\n",
     "Vectors of this sort are also called *standard basis* vectors.\n",
     "\n",
-    "For example, assuming that the system we have in mind is a bit, whose classical state set is therefore $\\{0,1\\}$, we denote\n",
+    "For example, assuming that the system we have in mind is a bit, the standard basis vectors are given by\n",
     "\n",
     "$$\n",
     "  \\vert 0\\rangle = \\begin{pmatrix}1\\\\0\\end{pmatrix}\n",
     "  \\quad\\text{and}\\quad\n",
     "  \\vert 1\\rangle = \\begin{pmatrix}0\\\\1\\end{pmatrix}.\n",
     "$$\n",
     "\n",
-    "Notice that any two-dimensional column vector can be expressed as a linear combination of these two vectors.\n",
+    "Notice that any probabilistic state can be expressed as a linear combination of these two vectors.\n",
```

Comment:
I'd rather that this is more general, referring to arbitrary vectors and not just probability vectors.